src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/RazorProjectHost.cs

```diff
@@ -188,7 +217,7 @@ protected override Task InitializeCoreAsync(CancellationToken cancellationToken)
             });
         }
 
-        private async Task<string> DetectFallbackLanguageVersion()
+        private async Task<RazorConfiguration> DetectFallbackLanguageConfiguration()
```

Comment:
This code only works right now when the project has already been restored. Currently trying to figure out how to deal with that. I was hoping that `GetResolvedReferencesAsync` would wait for a design-time build, but alas it does not.

This is only an issue for fallback cases.